products/osconfig/api.yaml

```diff
@@ -844,6 +844,41 @@ objects:
                    One day of the month. 1-31 indicates the 1st to the 31st day. -1 indicates the last day of the month.
                    Months without the target day will be skipped. For example, a schedule to run "every month on the 31st"
                    will not run in February, April, June, etc.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'rollout'
+        description: |
+          Rollout strategy of the patch job.
+        properties:
+          - !ruby/object:Api::Type::Enum
+            name: 'mode'
```

Comment:
I'd guess this should be required? Any reason a user would not want to specify the mode if they are specifying rollout?

Reply:
Yes it should